scripts/anaconda-import-initramfs-certs

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+# Transfers CA certificates imported in initramfs via kickstart
+# to anaconda environment after switchroot.
+
+# certificates dumped to the specified file are copied to root
+cp -rv /run/install/certificates/path/* /
```

Comment:
Is the `path` at the end really expected. It just feels like a leftover.

Also are we sure that this directory will exist in all the cases? I mean the service will fail if the path is missing, isn't it?

Reply:
> Is the `path` at the end really expected. It just feels like a leftover.
> 
It is [on purpose](https://github.com/rvykydal/anaconda/blob/e564a96e50b3b0d498bb2eb52c0cb7478dd044b9/dracut/parse-kickstart#L444), this subdirectory of `/run/install/certificates/path` should contain certificates that should be transported to switched root just by copying to the place, ie certificates only specified by `--dir` in kickstart. Other subdirectories can contain certificates of other types, for example `/run/install/certificates/category/global` would contain certificates with option `--category global` as in https://github.com/rvykydal/anaconda/commit/51c67c5a5b900b6b371d65ac3e125091580a2628#diff-572b90e7fb275a5450a384189ddd4651c02f9ab6059576366cdc25e5cdbec5f9R9. It is not so clear without the --category patch.

Reply:
> Also are we sure that this directory will exist in all the cases? I mean the service will fail if the path is missing, isn't it?

No it will not, good point, I'll perhaps modify copy to:
```
cp -rv /run/install/certificates/path/* / || true
```

Reply:
Updated